src/services/attendee/attendee-router.ts

```diff
@@ -150,4 +154,20 @@ attendeeRouter.get(
     }
 );
 
+// Get attendees based on a partial filter in body
+attendeeRouter.post(
+    "/filter",
+    RoleChecker([Role.Enum.ADMIN]),
+    async (req, res, next) => {
+        try {
+            const attendeeData = PartialAttendeeValidator.parse(req.body);
+            const attendees = await Database.ATTENDEE.find(attendeeData);
```

Comment:
return only the userId here (look into projection)

https://mongoosejs.com/docs/api/model.html#Model.find()